README.md

```diff
@@ -222,17 +219,29 @@ cd benckmark
 python ./kmeans.py
 ```
 
-to run a benchmark for different k-means implementations and print a short summary of the performance.
+to run a benchmark for different k-means implementations and print a short summary of
+the performance.
 
-Some parameters in the `__main__` section of the file `./benchmark/kmeans.py` are exposed for quick edition (`n_clusters`, `max_iter`, `skip_slow`, ...).
+Some parameters in the `__main__` section of the file `./benchmark/kmeans.py` are
+exposed for quick edition (`n_clusters`, `max_iter`, `skip_slow`, ...).
 
 ### Notes about the preferred floating point precision
 
-In many machine learning applications, operations using single-precision (float32) floating point data require twice as less memory that double-precision (float64), are regarded as faster, accurate enough and more suitable for GPU compute. Besides, most GPUs used in machine learning projects are significantly faster with float32 than with double-precision (float64) floating point data.
-
-To leverage the full potential of GPU execution, it's strongly advised to use a data loader that loads float32 data. By default, unless specified otherwise numpy array are created with type float64, so be especially careful to the type whenever the loader does not explicitly document the type nor expose a type option.
-
-Although it's less recommended to prevent avoidable data copies, it's also possible to transform float64 numpy arrays into float32 arrays using the [numpy.ndarray.astype](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.astype.html) type converter following this example:
+In many machine learning applications, operations using single-precision (float32)
+floating point data require twice as less memory that double-precision (float64), are
+regarded as faster, accurate enough and more suitable for GPU compute. Besides, most
+GPUs used in machine learning projects are significantly faster with float32 than with
+double-precision (float64) floating point data.
+
+To leverage the full potential of GPU execution, it's strongly advised to use a data
+loader that loads float32 data. By default, unless specified otherwise numpy array are
+created with type float64, so be especially careful to the type whenever the loader
+does not explicitly document the type nor expose a type option.
```

Comment:
```suggestion
To leverage the full potential of GPU execution, it's strongly advised to use a 
float32 data.

By default, unless specified otherwise numpy array are created with type float64,
so be especially careful to the type whenever the loader does not explicitly
document the type nor expose a type option.
```